### PR TITLE
docs: edits from joss review

### PR DESCRIPTION
**Pull request by rrsettgast**
- added links to input file for Tutorial 1
- added reference/link XML schema in Tutorial 1
- modified documentation for `NumericalMethods` section in Tutorial 1
- added note on future availability of permeability as a full tensor.
- added calls to `basicCleanup()` in `main()` for cases where it would be skipped.
- added note on the `Mandel` example for using the python plotting script.

### CODE REVIEW

File: src/docs/sphinx/tutorials/step01/Tutorial.rst [hunk 1]
```diff
@@ -91,6 +91,9 @@ in different regions of the domain at different moments of the simulation.
 
 In this first example, we use one type of solver in the entire domain and
 for the entire duration of the simulation.
+The input file for this tutorial can be found in the repository at
+`inputFiles/singlePhaseFlow/3D_10x10x10_compressible_smoke.xml`<https://github.com/GEOS-DEV/GEOS/blob/6dd40e776556ec1235ba183e00796f2eedc035ac/inputFiles/singlePhaseFlow/3D_10x10x10_compressible_smoke.xml>_, which also includes
```
Comment (jhuang2601) [inline src/docs/sphinx/tutorials/step01/Tutorial.rst]:
```suggestion
`inputFiles/singlePhaseFlow/3D_10x10x10_compressible_smoke.xml <https://github.com/GEOS-DEV/GEOS/blob/6dd40e776556ec1235ba183e00796f2eedc035ac/inputFiles/singlePhaseFlow/3D_10x10x10_compressible_smoke.xml>`_, which also includes
```

File: src/docs/sphinx/tutorials/step01/Tutorial.rst [hunk 2]
```diff
@@ -91,6 +91,9 @@ in different regions of the domain at different moments of the simulation.
...
+`inputFiles/singlePhaseFlow/3D_10x10x10_compressible_base.xml`<https://github.com/GEOS-DEV/GEOS/blob/6dd40e776556ec1235ba183e00796f2eedc035ac/inputFiles/singlePhaseFlow/3D_10x10x10_compressible_base.xml>_.
```
Comment (jhuang2601) [inline src/docs/sphinx/tutorials/step01/Tutorial.rst]:
```suggestion
`inputFiles/singlePhaseFlow/3D_10x10x10_compressible_base.xml <https://github.com/GEOS-DEV/GEOS/blob/6dd40e776556ec1235ba183e00796f2eedc035ac/inputFiles/singlePhaseFlow/3D_10x10x10_compressible_base.xml>`_.
```

File: src/docs/sphinx/tutorials/step01/Tutorial.rst [hunk 3]
```diff
@@ -134,7 +137,7 @@ In production runs, you may want to suppress most console output.
 
 For solvers of the ``SinglePhaseFVM`` family, one required attribute is a discretization scheme.
 Here, we use a Two-Point Flux Approximation (TPFA) finite volume discretization scheme called ``singlePhaseTPFA``.
-To know the list of admissible values of an attribute, please see GEOS's XML schema.
+To know the list of admissible values of an attribute, please see GEOS's XML schema (:doc:`docs/sphinx/CompleteXMLSchema`).
```
Comment (jhuang2601) [inline src/docs/sphinx/tutorials/step01/Tutorial.rst]:
```suggestion
To know the list of admissible values of an attribute, please see `GEOS's XML schema <https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/CompleteXMLSchema.html#>`_.
```

**Review by jhuang2601 — APPROVED** [review]

**Comment by timokoch** [comment]
This adds a note for Mandel's problem but as far as I can see this the same for all Python scripts. For example I randomly picked https://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoExtendedDruckerPrager/Example.html#a-comparison-between-geos-results-and-semi-analytical-results

**Comment by rrsettgast** [comment]
@timokoch Yes. I see there are 51 scripts in the repo that may have this issue:
```
~/Codes/geos/GEOS (feature/finalJOSSEdits ✗) grep -R ".. plot::" .     
./src/coreComponents/physicsSolvers/multiphysics/docs/gravityInducedStressInitialization/Example.rst:.. plot:: coreComponents/physicsSolvers/multiphysics/docs/gravityInducedStressInitialization/gravityInitializationFigure.py
./src/coreComponents/physicsSolvers/multiphysics/docs/userTableStressInitialization/Example.rst:.. plot:: coreComponents/physicsSolvers/multiphysics/docs/userTableStressInitialization/tableInitializationFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/thermoPoromechanics/thermalConsolidation/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/thermoPoromechanics/thermalConsolidation/thermalConsolidationPressureFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/thermoPoromechanics/thermalConsolidation/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/thermoPoromechanics/thermalConsolidation/thermalConsolidationTemperatureFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/thermoPoromechanics/thermalConsolidation/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/thermoPoromechanics/thermalConsolidation/thermalConsolidationDisplacementFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoModifiedCamClay/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoModifiedCamClay/TriaxialDriver_vs_SemiAnalytic_ViscoModifiedCamClay.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ExtendedDruckerPrager/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ExtendedDruckerPrager/TriaxialDriver_vs_SemiAnalytic_ExtendedDruckerPrager.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoDruckerPrager/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoDruckerPrager/TriaxialDriver_vs_SemiAnalytic_ViscoDruckerPrager.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/DruckerPrager/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/DruckerPrager/TriaxialDriver_vs_SemiAnalytic_DruckerPrager.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ModifiedCamClay/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ModifiedCamClay/TriaxialDriver_vs_SemiAnalytic_ModifiedCamClay.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/RelaxationTest/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/RelaxationTest/relaxationTestFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoExtendedDruckerPrager/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/viscoplasticity/ViscoExtendedDruckerPrager/TriaxialDriver_vs_SemiAnalytic_ViscoExtendedDruckerPrager.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/pennyFracToughnessDominated/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/pennyFracToughnessDominated/pennyShapedToughnessDominatedFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/pknFracViscosityDominated/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/pknFracViscosityDominated/pknViscosityDominatedFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdValidation/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdValidation/kgdValidationFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdViscosityDominated/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdViscosityDominated/kgdViscosityDominatedFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdToughnessDominated/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/kgdToughnessDominated/kgdToughnessDominatedFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/proppantSlotTest/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/proppantSlotTest/proppantSlotTestFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/pennyFracViscosityDominated/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/hydraulicFracture/pennyFracViscosityDominated/pennyShapedViscosityDominatedFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/carbonStorage/buckleyLeverett/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/carbonStorage/buckleyLeverett/buckleyLeverettFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/carbonStorage/isothermalLeakyWell/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/carbonStorage/isothermalLeakyWell/isothermalLeakyWellFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/carbonStorage/thermalLeakyWell/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/carbonStorage/thermalLeakyWell/thermalLeakyWell.py
./src/docs/sphinx/advancedExamples/validationStudies/faultMechanics/singleFracCompression/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/faultMechanics/singleFracCompression/singleFracCompressionFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/faultMechanics/intersectFrac/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/faultMechanics/intersectFrac/intersectFracFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/faultMechanics/sneddon/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/faultMechanics/sneddon/sneddonFigure.py    
./src/docs/sphinx/advancedExamples/validationStudies/faultMechanics/faultVerification/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/faultMechanics/faultVerification/faultVerificationFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/poromechanics/mandel/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/poromechanics/mandel/mandelFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_temperature.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_displacement.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedThermoElasticWellbore/thermoElastic_casedWellbore_stress.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedElasticWellbore/casedElasticWellboreFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentVolumetricHeatCapacity/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentVolumetricHeatCapacity/temperatureDependentVolumetricHeatCapacity_plot.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/THM_wellbore_temperature_pressure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/THM_wellbore_displacement.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/thermoPoroElasticWellbore/THM_wellbore_stress.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/thermoElastic_casedContactWellbore_temperature.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/thermoElastic_casedContactWellbore_displacement.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactThermoElasticWellbore/thermoElastic_casedContactWellbore_stress.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/kirschWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/kirschWellbore/kirschWellboreFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/dpWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/dpWellbore/dpWellbore_plot.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/deviatedPoroElasticWellbore/Example2.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/deviatedPoroElasticWellbore/deviatedPoroElasticWellboreExample2Figure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/deviatedPoroElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/deviatedPoroElasticWellbore/deviatedPoroElasticWellboreExample1Figure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/linearThermalDiffusion/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/linearThermalDiffusion/linearThermalDiffusion_plot.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/mccWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/mccWellbore/mccWellboreFigure.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/edpWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/edpWellbore/edpWellbore_plot.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/casedContactElasticWellbore/elastic_casedWellbore_displacementJump.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentSinglePhaseThermalConductivity/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentSinglePhaseThermalConductivity/temperatureDependentSinglePhaseThermalConductivity_plot.py
./src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/deviatedElasticWellbore/Example.rst:.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/deviatedElasticWellbore/deviatedElasticWellboreFigure.py
./src/docs/sphinx/basicExamples/triaxialDriver/Example.rst:.. plot:: docs/sphinx/basicExamples/triaxialDriver/triaxialDriverFigure.py
./src/docs/sphinx/basicExamples/poromechanics/Example.rst:.. plot:: docs/sphinx/basicExamples/poromechanics/poromechanicsFigure.py
./src/docs/sphinx/basicExamples/multiphaseFlowWithWells/Example.rst:.. plot:: docs/sphinx/basicExamples/multiphaseFlowWithWells/multiphaseFlowWithWellsFigure.py
```

@jhuang2601 I think the issue here is that the python scripts are hard coded for our sphinx documentation. We need to:

- make all of these scripts generic s.t. they take:
  - takes the xml path/s as arguments,
  - take the output dir path as an argument
- add the hardcoded sphinx doc paths from the call location...and hope that works...or...
- create a dummy script that calls the generic script with the hardcoded paths.

**Review by CusiniM — APPROVED** [review]

**Pull request merged**